addons/attributes/XEH_preInit.sqf

```diff
@@ -6,17 +6,45 @@ PREP_RECOMPILE_START;
 #include "XEH_PREP.hpp"
 PREP_RECOMPILE_END;
 
+#include "initKeybinds.sqf"
+
 // Namespace thats stores data of all attribute displays
 GVAR(displays) = [] call CBA_fnc_createNamespace;
 
 // Namespace that tracks selected marker colors by marker type
 // Colors are applied to newly placed markers of the same type
 GVAR(previousMarkerColors) = [] call CBA_fnc_createNamespace;
 
+GVAR(selectedGroup) = grpNull;
+GVAR(selectedEntity) = objNull;
+
 ["ModuleCurator_F", "Init", {
     params ["_logic"];
 
     _logic addEventHandler ["CuratorMarkerPlaced", {call FUNC(handleMarkerPlaced)}];
+
+    // handle group being select when all of its sub entities are selected
+    private _fnc_checkGroupSelection = {
+        private _entity = _this # 1;
```

Comment:
Afaik, we don't use `#`.

Reply:
https://github.com/ampersand38/ZEN/commit/35b77767cd0d95abd8411a11155d093cd301648c

Reply:
You don’t need to attach these commit info, just mark it solved otherwise a lot of spam emails :)